lib/streamlit/type_util.py

```diff
@@ -338,6 +370,13 @@ def is_keras_model(obj: object) -> bool:
     )
 
 
+def is_list_of_scalars(data: Iterable[Any]) -> bool:
+    """Check if the list only contains scalar values."""
+    # Overview on all value that are interpreted as scalar:
+    # https://pandas.pydata.org/docs/reference/api/pandas.api.types.is_scalar.html
+    return infer_dtype(data, skipna=True) not in ["mixed", "unknown-array"]
```

Comment:
nit: `skipna=True` is defaulted to true. 
![Uploading Screenshot 2023-01-23 at 10.47.33 PM.png…]()

Reply:
I think I put this to true since we will otherwise get a warning message in older pandas versions. The screenshot you attached did not correctly upload, what does it show?